SW_Model.c

```diff
@@ -48,13 +48,18 @@
 #include "SW_SoilWater.h"  /* for setup_new_year() */
 #include "SW_Times.h"
 #include "SW_Model.h"
+#include "pcg/pcg_basic.h"
 
 /* =================================================== */
 /*                  Global Variables                   */
 /* --------------------------------------------------- */
 extern SW_SITE SW_Site; /* for reset attribute */
 SW_MODEL SW_Model; /* declared here, externed elsewhere */
 
+#ifndef STEPWAT

```

Comment:
Why wrap this in `#ifndef STEPWAT`? -- this is confusing; extern `markov_rng` on the STEPWAT2 side if you need it there instead of two different places to define this variable.